apps/contracts/contracts/IZKGroups.sol

```diff
@@ -1,16 +1,13 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.4;
 
-/// @title zk-groups interface.
-/// @dev Interface of a ZKGroups contract.
+/// @title ZKGroups contract interface.
 interface IZKGroups {
-    struct Verifier {
-        address contractAddress;
-        uint256 merkleTreeDepth;
-    }
+    error ZKGroups__GroupDoesNotExist();
+    error ZKGroups__YouAreUsingTheSameNillifierTwice();
+    error ZKGroups__MerkleTreeDepthIsNotSupported();
 
-    struct OffchainGroup {
-        bytes32 groupName;
+    struct Group {
```

Comment:
Should we call this `GroupData`, `GroupConfig` or something similar?
Because `mapping(bytes32 => Group)` variable is called `groups` 🤔